requirements.txt

```diff
@@ -0,0 +1,8 @@
+cycler==0.10.0
```

Comment:
I am reading everywhere that requirements.txt are not needed?
https://medium.com/@boscacci/why-and-how-to-make-a-requirements-txt-f329c685181e
When is this file used really? If it is only for developers, we need to hide it better somewhere in the directories.

Reply:
requirements.txt are for developpers. not for the actual package, as i said in the commit comment

Reply:
I don,t really see anywhere to place it. It has always been in the main folder everytime I have seen projects with people working with venv. It seems that making a folder specifically for this file might be too much.

Reply:
or you build your dev environment with pip install raytracing (installs dependencies) then pip uninstall raytracing

Reply:
yeah, I think this is not really a dev environnement issue since it has only matplotlib and numpy,  but i don't understand the angst against venv or the need to hide requirements.txt, since it is common practice, which is the proper method i believe.

Reply:
Nonehteless, I'll remove the file.